securityhub-forwarder/sam/template.yaml

```diff
@@ -45,7 +45,7 @@ Resources:
     Type: 'AWS::Serverless::Function'
     Properties:
       Handler: securityhub_forwarder.lambda_handler
-      Runtime: python3.7
+      Runtime: python3.11
       Layers:
         - !Sub 'arn:aws:lambda:${AWS::Region}:956882708938:layer:securityhub_deps:1'
       CodeUri: ../src/
```

Comment:
Where is this code located? Is it checked-in to some repo and/ or s3 bucket?
If yes can you please share the code location path.
It is recommended to use S3 path in codeUri for future maintenance purposes.

Reply:
Only version update